src/hooks/useOrgHooks.js

```diff
@@ -21,8 +21,6 @@ import {
 
 // Organzation
 const ORG_ADDRESS = '0xe9869a0bbc8fb8c61b7d81c33fa2ba84871b3b0e'
-const ORG_SUBRAPH_URL =
-  'https://api.thegraph.com/subgraphs/name/1hive/aragon-xdai'
```

Comment:
This is not needed since [Connect knows about xDai](https://github.com/aragon/connect/blob/master/packages/connect-tokens/src/thegraph/connector.ts#L16-L18), and it will also allow Connect to pick the right one when started with another network ([e.g. `4`](https://github.com/1Hive/honey-pot/blob/fd4fabb1928dac344a0c0ccdcf4763639fd23fce/package.json#L27)).

Reply:
so you guys deployed an aragon xdai subgraph? because here we are using the one under `/1hive`

Reply:
Oh you’re right sorry, I checked too quickly and thought it was the same URL. I don’t know if there is a difference between the two. @0xGabi do you think we could use the Aragon one here?

Reply:
Actually I configured it to use the one deployed by 1Hive 👍